### ISSUE TITLE
CTRL + C not working on Windows

### ISSUE THREAD

**Issue by MidSpike** [kufii/CodeSnap#66]
Doing CTRL + C does not copy the image to the clipboard, neither does clicking on the shutter button.

System Specs:
- Windows 10 Pro 20H2 (19042.928)
- Visual Studio Code: v1.55.2

**Comment by ourainbows** [comment]
Yeah; I try with codesnap.shutterAction: copy but it doesn't work.

**Comment by PrincessAkira** [comment]
can also confirm, it doesnt wanna work

**Comment by srodriguez20** [comment]
yeah, It doesn't work, it's coping a file without extension in my case

**Comment by emilioestebanez** [comment]
I had to downgrade to 1.2.1 to get it working again for the time being

**Comment by DGamer007** [comment]
When I press Ctrl+C , this extention is just copying the text in clipboard and  I dont know how but I am not even able to paste that text in any program.

**Comment by DGamer007** [comment]
Has the same problem ,but I have noticed that when I press `Ctrl + C` , this extention does copy **only text** to clipboard and also I am not even able to paste that **only text** anywhere in my pc.

**Comment by PrincessAkira** [comment]
How 2 downgrade though?

**Comment by DGamer007** [comment]
In that extension tab you will find a uninstall button and at the end of that button you'll find down-aero , click on that down-aero,
click on install another version
and then pick your version from list

**Comment by Jaeiya** [comment]
Can confirm that the copy to clipboard function is broken in all CodeSnap versions after `1.2.1`.

**Comment by MuTsunTsai** [comment]
I have confirmed that the problem is caused by one of the dependencies, `img-clipboard`. By downgrading `img-clipboard` from v1.1.0 back to v1.0.4, the problem is solved. I'll make a PR for this.

**Comment by MuTsunTsai** [comment]
I later found a completely different method of copying the image, without using `img-clipboard` (sorry about that @kufii). I've just made a PR #71.

**Comment by MuTsunTsai** [comment]
Confirmed that this issue is fixed in v1.3.4
